### PR TITLE
Feature 62 clear shortlist confirm

### LINKED ISSUE #62: add "confirm" option to clear shortlist button
There should be a confirmation button or modal before the shortlist gets wiped.

### PR DESCRIPTION
**Pull request by Neil511**
fixes #62

### CODE REVIEW

File: js/content_script.js [hunk 1]
```diff
@@ -450,6 +450,23 @@ function removeFromShortlistCall(jobIds, action) {
...
+		var callback = event.data.callback;
+
+		Notify.confirm({
+				'title': message,
+				'left': 'No',
+				'right': 'Yes',
+				'class': '',
+				modal: true,
+				fn: function(e) {
+						console.log(e);
```
Comment (peygar) [inline js/content_script.js]: are we keeping this log or was it for debugging?

File: manifest.json [hunk 2]
```diff
@@ -26,6 +26,7 @@
       ],
       "js": [
         "js/external/jquery.min.js",
+        "js/external/simple-modal.js",
```
Comment (peygar) [inline manifest.json]: `external/*`?

File: js/content_script.js [hunk 3]
```diff
@@ -450,6 +450,23 @@ function removeFromShortlistCall(jobIds, action) {
     }
 
 }
+function confirmDialog(event) {
+		var message = event.data.message;
+		var callback = event.data.callback;
```
Comment (peygar) [inline js/content_script.js]: I'd maybe name it `confirmCallback` for clarity

File: js/external/simple-modal.js [hunk 4]
```diff
@@ -0,0 +1,280 @@
+var util = {
```
Comment (gurpreetg) [inline js/external/simple-modal.js]: Can we not get a minified version for this? If not, lets use https://jscompress.com/ to compress this.

File: manifest.json (hunk 2 above)
Comment (gurpreetg) [reply manifest.json]: @peygar I think I tried doing that few months ago and had some problems. But @Neil511 can try doing it and see if it works. Make sure to test it properly tho!

File: js/content_script.js (hunk 1 above)
Comment (gurpreetg) [reply js/content_script.js]: good job @Neil511

File: js/external/simple-modal.js (hunk 4 above)
Comment (Neil511) [reply js/external/simple-modal.js]: We could but I thought we'd be changing the styles frequently or reusing them elsewhere so for now Ieft it as is.

File: manifest.json (hunk 2 above)
Comment (Neil511) [reply manifest.json]: Tried and failed, apparently it won't allow it: https://stackoverflow.com/questions/36651639/chrome-extension-manifest-json-load-all-js-in-folder

Comment (peygar) [reply manifest.json]: feels bad man

**Pull request merged**